edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -45,27 +57,56 @@ During this process, the {product-title} installation program installs {op-syste
 
-The following content describes the components in an image-based installation and deployment.
+* The `install-config.yaml` installation manifest.
 
-Seed image:: OCI container image generated from a dedicated cluster with the target {product-title} version. 
+* The `image-based-config.yaml` manifest.
 
-Seed cluster:: Dedicated {sno} cluster that you use to create a seed image and is deployed with the target {product-title} version.
+The `openshift-install` installation program consumes these resources to generate a configuration ISO that you can attach to your preinstalled target host and complete the deployment.
 
-{lcao}:: Generates the seed image, embeds the seed image URL in the live installation ISO, and reconfigures the host during remote site deployment.
+== Image-based installation and deployment key components
 
-`openshift-install` installation program:: Use the `openshift-install` program to manually create a configuration ISO. Attach the configuration ISO to a preinstalled host to complete the deployment.
+The following content describes key components in image-based installation and deployments.
+
+Seed image:: A seed image is an OCI container image generated from a dedicated cluster with the target {product-title} version. 
+
+Seed cluster:: A seed cluster is a dedicated {sno} cluster that you use to create a seed image and is deployed with the target {product-title} version.
+
+{lcao}:: In image-based installations, you can use the {lcao} to generate a seed image, embed the seed image URL in a live installation ISO, and reconfigure the host during a deployment at a remote site.
+
+Image Based Install (IBI) Operator:: When you deploy managed clusters, the IBI Operator creates a configuration ISO from the site-specific resources you define in the hub cluster, and attaches the configuration ISO to the preinstalled host by using a bare-metal provisioning service.
+
+{product-title} installation program:: You can use the `openshift-install` installation program to manually create a configuration ISO. Attach the configuration ISO to a preinstalled host to complete the deployment.
 
 include::modules/ibi-image-based-install-cluster-guide.adoc[leveloffset=+1]
 
 include::modules/ibi-validated-software-versions.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/about/welcome-to-red-hat-advanced-cluster-management-for-kubernetes#multicluster-architecture[Multicluster architecture]
+
+* link:https://docs.openshift.com/container-platform/4.16/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.html[Understanding the image-based upgrade for {sno} clusters]
```

Comment:
@eranco74 Is there a reason that 4.16 is referenced?

Reply:
Nope, the initial TP was 4.16, I think this should reference the version matching the current docs